kindtest.sh

```diff
@@ -0,0 +1,102 @@
+#!/bin/bash
```

Comment:
copyright notice

Reply:
Ugh, okay... that's embarrassing. Thanks.

Reply:
2019?

Reply:
More coffee needed... Thanks for having my back.  I'm not with it today.